src/exchangeRateRouter.ts

```diff
@@ -18,14 +26,51 @@ const asExchangeRatesReq = asObject({
 
 export type ExchangeRateReq = ReturnType<typeof asExchangeRateReq>
 
+const { couchUri, fiatCurrencyCodes: FIAT_CODES } = config
 const EXCHANGE_RATES_BATCH_LIMIT = 100
 
-const nanoDb = nano(config.couchUri)
+const nanoDb = nano(couchUri)
 const dbRates: nano.DocumentScope<DbDoc> = nanoDb.db.use('db_rates')
 promisify(dbRates)
 
+// *** UTILS ***
+
+const addIsoToMaybeFiatCode = (code: string): string =>
+  FIAT_CODES.includes(addIso(code)) ? addIso(code) : code
```

Comment:
Hmm, this may be clearer with a comment as to why we need to do the lookup in the config array. Isn't there a `isFiatCode` in utils which can be used instead of a lookup in the `config.fiatCurrencyCodes` array? A comment would be nice to explain what this function does and why; when do we add `iso:` vs when we dont? Or, perhaps a rename might help better reflect what the function does.